README.md

```diff
@@ -43,19 +43,18 @@ You will be prompted to install the Python dependencies during the first use.
 - _**[Python 3.7.4](https://www.python.org/downloads/)**_: Make sure you've added python and pip to your PATH in your environment variables. (1)
 - _**[Python VS Code extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python)**_: This will be installed automatically from the marketplace when you install Device Simulator Express.
 
-The following dependecies can be installed for you by the extension by clicking yes when you are prompted to (**except** `pywin32` which is needed only on Windows platform). (2)
-
-- _**Playsound**_  
-  install by typing the following commands in a console: `pip install playsound`
-
-- _**Pywin 32**_  
-  install by typing the following commands in a console (only for Windows computers, you must run it manually): `pip install pywin32`
-- _**Python-Socketio**_  
-   install by typing the following commands in a console: `pip install python-socketio`
-- _**Requests**_  
-   install by typing the following commands in a console: `pip install requests`
-- _**Application Insights**_  
-  install by typing the following commands in a console: `pip install applicationinsights`
+- Python Modules for Deploying to Device
+    - If you're on Windows, use the following command in the console to install pywin32: `pip install pywin32`
+
+- Python Modules for Simulation
+  - **Note:** On extension activation, you will be prompted with a popup message asking if you want the modules to be automatically installed for you. The following Python modules should be downloaded when you select "yes" on the prompt message. **If modules are not installed correctly, please use the "pip install" commands listed below.**

```

Comment:
should this note be at the top/or placed somewhere else because it also applies to "Deploying to Device"

Reply:
Just a suggestion on how we can address this, instead of having these two topics, we can have something along these lines:
(I guess the same goes for the text on Install.md and developers-setup.md)

> - Python Modules for Deploying to Device
> - Python Modules for Simulation

We can just have:
> - **(Notes here)**
>   - Python Modules
>       - Pywin32 (only windows etc)
> **Running the simulator:**
>       - (list extra modules here)

Reply:
I tried to make it clearer with the newest commit. Is it a bit more organized now?